packages/twenty-front/src/modules/object-metadata/utils/applyQuotesToString.ts

```diff
@@ -0,0 +1,3 @@
+export const applyQuotesToString = (defaultValue: string) => {
```

Comment:
I feel like defaultValue could have a more generic name since the util seems generic enough